### ISSUE TITLE
Request: list views

### ISSUE THREAD

**Issue by ctoth** [andlabs/libui#44]
Libui is an awesome project that makes it much easier to make natively accessible applications on various platforms.
Currently it's impossible to render a list of items in a list view.

There are two pieces of behavior that I see as important for a list view implementation (wx fails onthe second on Mac):
1: Displaying a fixed list of columnar data
2: Displaying a list of items dynamically fetched from a datasource

Here is documentation on the underlying UI controls:
On windows: https://msdn.microsoft.com/en-us/library/windows/desktop/bb774737%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396

On mac: https://developer.apple.com/library/mac/documentation/Cocoa/Reference/ApplicationKit/Classes/NSTableView_Class/

**Comment by andlabs** [comment]
Yes, that is all planned for after the next packaged release. There's something in `_wip` about it too.

**Comment by iKlsR** [comment]
I was about to ask about tables but hopefully that's up next it seems.

**Comment by TravisJoe** [comment]
That is great to hear. One of the main features I need it columnar data display for testing and monitoring applications that update reasonably quickly. This sounds like it might meet that need.

**Comment by jreinert** [comment]
:+1:

**Comment by nus** [comment]
+1

**Comment by leyafo** [comment]
+1

**Comment by leyafo** [comment]
Has it anything I can help?

**Comment by bcampbell** [comment]
see also issue #159
